sos/plugins/pcp.py

```diff
@@ -148,10 +148,8 @@ def setup(self):
             os.path.join(self.pcp_log_dir, '*/*/config*')
         ])
 
-        # Need to get the current status of the PCP infrastructure
-        self.add_cmd_output("pcp")
         # Collect a summary for the current day
-        res = self.get_command_output('pcp')
+        res = self.get_cmd_output_now('pcp')
```

Comment:
with `save=False`?

Reply:
We previously ran `add_cmd_output('pcp')`, which would be redundant with the new approach of `get_cmd_output_now()`. So  `save=False` would be a regression here.